src/traversals/bfs.jl

```diff
@@ -189,12 +189,15 @@ function is_bipartite(g::SimpleGraph, s::Int)
     return visitor.is_bipartite
 end
 
-function is_bipartite(g::SimpleGraph)
-    nvg = nv(g)
-    for v in vertices(g)
-        if !is_bipartite(g, v)
-            return false
+if VERSION >= v"0.4.0-dev+6371"
+    is_bipartite(g::SimpleGraph) = all([is_bipartite(g,v) for v in vertices(g)])
```

Comment:
One (weak) reason not to do this is coverage: we will no longer get a report of when this returns true. It looks cleaner to me, but on the other hand you have to keep the 0.3 version around... Up to you!